### DISCUSSION TITLE
转转二手交易平台

### DISCUSSION THREAD

**Discussion by LesVagues** [share121/inter-knot#2233] [其他]
这么流畅的操作肯定离不开一台好用的设备，多亏了我在转转上新买的二手OPPO A5比新机便宜了两块，省下的钱都可以买公交车票了，转转是一个官方验证二手平台，平时不用的旧手机、旧平板，我都放转转上回收了，下单就是人来上门回收，我那台旧手机回收了2¥多，价格满意现场打钱，推荐你们用转转更省钱。
![2ae7db2e-d2cf-4f80-8b46-e52b8719c42c](https://github.com/user-attachments/assets/5c9d0c64-abef-415c-ab3d-8126eaa77e70)

**Comment by LesVagues** [comment]
JUST 玩梗 XDM，别举报我

**Comment by Genshin-Neuvillette** [comment]
（music~）拼多多，拼多多，拼得多，赚的多~

**Comment by break-under-star** [comment]
怎么广告都打到绳网来了，不怕治安局？（怒）

**Comment by Rain-YYC** [comment]
这委托.....

**Comment by KUYUE404** [comment]
这软件在治安局都没有报备，各位绳匠，小心诈骗！<img src="https://img-static.mihoyo.com/communityweb/upload/1df6c0b9d320b28196682fd4e9798e75.gif" alt="微笑面对" width="75" >

**Comment by shuiyuan2023** [comment]
到绳网了还是逃不过吗

**Comment by 206288602** [comment]
话说是不是应该缴广告费<img src="https://bbs-static.miyoushe.com/static/2024/01/31/cf527198a404a4db566dc4a9d68befc3_2947652818567379105.png" alt="花火 嘿嘿" width="75" >

**Comment by Random114** [comment]
这委托我接了

**Comment by MC-FanBuFan** [comment]
这啥玩意？

**Comment by Momsaysnamesthataretoolongmakefoolslook** [comment]
不是转转怎么转到绳网了
